test/server.ts

```diff
@@ -78,5 +87,57 @@ describe('Onboard server API', function () {
         birthDate: new Date(userInput.birthDate).getTime().toString(),
       });
     });
+
+    it('should fail to create user with repeated e-mail', async function () {
+      await prisma.user.create({
+        data: {
+          name: userInput.name,
+          email: userInput.email,
+          password: userInput.password,
+          birthDate: new Date(userInput.birthDate),
+        },
+      });
+      const previousUserCount = await prisma.user.count();
+      const response = await axios.post(url, createUserMutation);
+      const currentUserCount = await prisma.user.count();
+
+      expect(currentUserCount).to.be.eq(previousUserCount);
+      expect(response.data.data.createUser).to.be.eq(null);
+      expect(response.data.errors[0].message).to.be.eq('E-mail is already in use.');
```

Comment:
Sugeriria sempre tentar utilizar `to.be.deep.eq` no body completo da resposta, porque assim você testa se todos os campos do body estão de acordo com o que deveriam, e até se não faltaram ou ou se não tem campos a mais.